In [1]:
import pandas as pd

In [2]:
df = pd.read_csv('https://raw.githubusercontent.com/datasciencedojo/datasets/master/titanic.csv')

In [5]:
df[df['Fare'] > 200]

PassengerId  Survived  Pclass  \
27            28         0       1   
88            89         1       1   
118          119         0       1   
258          259         1       1   
299          300         1       1   
311          312         1       1   
341          342         1       1   
377          378         0       1   
380          381         1       1   
438          439         0       1   
527          528         0       1   
557          558         0       1   
679          680         1       1   
689          690         1       1   
700          701         1       1   
716          717         1       1   
730          731         1       1   
737          738         1       1   
742          743         1       1   
779          780         1       1   

                                                  Name     Sex   Age  SibSp  \
27                      Fortune, Mr. Charles Alexander    male  19.0      3   
88                          Fortune, Miss. Mabel Helen  female  23.0      3   
118                           Baxter, Mr. Quigg Edmond    male  24.0      0   
258                                   Ward, Miss. Anna  female  35.0      0   
299    Baxter, Mrs. James (Helene DeLaudeniere Chaput)  female  50.0      0   
311                         Ryerson, Miss. Emily Borie  female  18.0      2   
341                     Fortune, Miss. Alice Elizabeth  female  24.0      3   
377                          Widener, Mr. Harry Elkins    male  27.0      0   
380                              Bidois, Miss. Rosalie  female  42.0      0   
438                                  Fortune, Mr. Mark    male  64.0      1   
527                                 Farthing, Mr. John    male   NaN      0   
557                                Robbins, Mr. Victor    male   NaN      0   
679                 Cardeza, Mr. Thomas Drake Martinez    male  36.0      0   
689                  Madill, Miss. Georgette Alexandra  female  15.0      0   
700  Astor, Mrs. John Jacob (Madeleine Talmadge Force)  female  18.0      1   
716                      Endres, Miss. Caroline Louise  female  38.0      0   
730                      Allen, Miss. Elisabeth Walton  female  29.0      0   
737                             Lesurer, Mr. Gustave J    male  35.0      0   
742              Ryerson, Miss. Susan Parker "Suzette"  female  21.0      2   
779  Robert, Mrs. Edward Scott (Elisabeth Walton Mc...  female  43.0      0   

     Parch    Ticket      Fare            Cabin Embarked  
27       2     19950  263.0000      C23 C25 C27        S  
88       2     19950  263.0000      C23 C25 C27        S  
118      1  PC 17558  247.5208          B58 B60        C  
258      0  PC 17755  512.3292              NaN        C  
299      1  PC 17558  247.5208          B58 B60        C  
311      2  PC 17608  262.3750  B57 B59 B63 B66        C  
341      2     19950  263.0000      C23 C25 C27        S  
377      2    113503  211.5000              C82        C  
380      0  PC 17757  227.5250              NaN        C  
438      4     19950  263.0000      C23 C25 C27        S  
527      0  PC 17483  221.7792              C95        S  
557      0  PC 17757  227.5250              NaN        C  
679      1  PC 17755  512.3292      B51 B53 B55        C  
689      1     24160  211.3375               B5        S  
700      0  PC 17757  227.5250          C62 C64        C  
716      0  PC 17757  227.5250              C45        C  
730      0     24160  211.3375               B5        S  
737      0  PC 17755  512.3292             B101        C  
742      2  PC 17608  262.3750  B57 B59 B63 B66        C  
779      1     24160  211.3375               B3        S

In [9]:
# 15 People is travelling with zero fare
len(df[df['Fare'] <= 0])

15

In [10]:
len(df[df['Fare'] <= 10])

336

In [24]:
df[(df['Fare'] > 0) & (df['Fare'] <= 10) ]

PassengerId  Survived  Pclass                            Name     Sex  \
0              1         0       3         Braund, Mr. Owen Harris    male   
2              3         1       3          Heikkinen, Miss. Laina  female   
4              5         0       3        Allen, Mr. William Henry    male   
5              6         0       3                Moran, Mr. James    male   
12            13         0       3  Saundercock, Mr. William Henry    male   
..           ...       ...     ...                             ...     ...   
877          878         0       3            Petroff, Mr. Nedelio    male   
878          879         0       3              Laleff, Mr. Kristo    male   
881          882         0       3              Markun, Mr. Johann    male   
884          885         0       3          Sutehall, Mr. Henry Jr    male   
890          891         0       3             Dooley, Mr. Patrick    male   

      Age  SibSp  Parch            Ticket    Fare Cabin Embarked  
0    22.0      1      0         A/5 21171  7.2500   NaN        S  
2    26.0      0      0  STON/O2. 3101282  7.9250   NaN        S  
4    35.0      0      0            373450  8.0500   NaN        S  
5     NaN      0      0            330877  8.4583   NaN        Q  
12   20.0      0      0         A/5. 2151  8.0500   NaN        S  
..    ...    ...    ...               ...     ...   ...      ...  
877  19.0      0      0            349212  7.8958   NaN        S  
878   NaN      0      0            349217  7.8958   NaN        S  
881  33.0      0      0            349257  7.8958   NaN        S  
884  25.0      0      0   SOTON/OQ 392076  7.0500   NaN        S  
890  32.0      0      0            370376  7.7500   NaN        Q  

[321 rows x 12 columns]

In [25]:
# Fare --> 1-10 --> 321
len(df[(df['Fare'] > 0) & (df['Fare'] <= 10) ])

321

In [26]:
len(df[(df['Fare'] > 10) & (df['Fare'] <= 20) ])

179

In [27]:
len(df[(df['Fare'] > 20) & (df['Fare'] <= 30) ])

142

In [28]:
len(df[(df['Fare'] > 30) & (df['Fare'] <= 40) ])

58

In [29]:
df['Pclass'].value_counts()

Pclass
3    491
1    216
2    184
Name: count, dtype: int64

In [31]:
df[df['Pclass'] == 3]

PassengerId  Survived  Pclass                                      Name  \
0              1         0       3                   Braund, Mr. Owen Harris   
2              3         1       3                    Heikkinen, Miss. Laina   
4              5         0       3                  Allen, Mr. William Henry   
5              6         0       3                          Moran, Mr. James   
7              8         0       3            Palsson, Master. Gosta Leonard   
..           ...       ...     ...                                       ...   
882          883         0       3              Dahlberg, Miss. Gerda Ulrika   
884          885         0       3                    Sutehall, Mr. Henry Jr   
885          886         0       3      Rice, Mrs. William (Margaret Norton)   
888          889         0       3  Johnston, Miss. Catherine Helen "Carrie"   
890          891         0       3                       Dooley, Mr. Patrick   

        Sex   Age  SibSp  Parch            Ticket     Fare Cabin Embarked  
0      male  22.0      1      0         A/5 21171   7.2500   NaN        S  
2    female  26.0      0      0  STON/O2. 3101282   7.9250   NaN        S  
4      male  35.0      0      0            373450   8.0500   NaN        S  
5      male   NaN      0      0            330877   8.4583   NaN        Q  
7      male   2.0      3      1            349909  21.0750   NaN        S  
..      ...   ...    ...    ...               ...      ...   ...      ...  
882  female  22.0      0      0              7552  10.5167   NaN        S  
884    male  25.0      0      0   SOTON/OQ 392076   7.0500   NaN        S  
885  female  39.0      0      5            382652  29.1250   NaN        Q  
888  female   NaN      1      2        W./C. 6607  23.4500   NaN        S  
890    male  32.0      0      0            370376   7.7500   NaN        Q  

[491 rows x 12 columns]

In [32]:
df[(df['Pclass'] == 3)  & (df['Survived']==1)]

PassengerId  Survived  Pclass  \
2              3         1       3   
8              9         1       3   
10            11         1       3   
19            20         1       3   
22            23         1       3   
..           ...       ...     ...   
838          839         1       3   
855          856         1       3   
858          859         1       3   
869          870         1       3   
875          876         1       3   

                                                  Name     Sex   Age  SibSp  \
2                               Heikkinen, Miss. Laina  female  26.0      0   
8    Johnson, Mrs. Oscar W (Elisabeth Vilhelmina Berg)  female  27.0      0   
10                     Sandstrom, Miss. Marguerite Rut  female   4.0      1   
19                             Masselmani, Mrs. Fatima  female   NaN      0   
22                         McGowan, Miss. Anna "Annie"  female  15.0      0   
..                                                 ...     ...   ...    ...   
838                                    Chip, Mr. Chang    male  32.0      0   
855                         Aks, Mrs. Sam (Leah Rosen)  female  18.0      0   
858              Baclini, Mrs. Solomon (Latifa Qurban)  female  24.0      0   
869                    Johnson, Master. Harold Theodor    male   4.0      1   
875                   Najib, Miss. Adele Kiamie "Jane"  female  15.0      0   

     Parch            Ticket     Fare Cabin Embarked  
2        0  STON/O2. 3101282   7.9250   NaN        S  
8        2            347742  11.1333   NaN        S  
10       1           PP 9549  16.7000    G6        S  
19       0              2649   7.2250   NaN        C  
22       0            330923   8.0292   NaN        Q  
..     ...               ...      ...   ...      ...  
838      0              1601  56.4958   NaN        S  
855      1            392091   9.3500   NaN        S  
858      3              2666  19.2583   NaN        C  
869      1            347742  11.1333   NaN        S  
875      0              2667   7.2250   NaN        C  

[119 rows x 12 columns]

In [33]:
df[(df['Pclass'] == 3)  & (df['Survived']==0)]

PassengerId  Survived  Pclass                                      Name  \
0              1         0       3                   Braund, Mr. Owen Harris   
4              5         0       3                  Allen, Mr. William Henry   
5              6         0       3                          Moran, Mr. James   
7              8         0       3            Palsson, Master. Gosta Leonard   
12            13         0       3            Saundercock, Mr. William Henry   
..           ...       ...     ...                                       ...   
882          883         0       3              Dahlberg, Miss. Gerda Ulrika   
884          885         0       3                    Sutehall, Mr. Henry Jr   
885          886         0       3      Rice, Mrs. William (Margaret Norton)   
888          889         0       3  Johnston, Miss. Catherine Helen "Carrie"   
890          891         0       3                       Dooley, Mr. Patrick   

        Sex   Age  SibSp  Parch           Ticket     Fare Cabin Embarked  
0      male  22.0      1      0        A/5 21171   7.2500   NaN        S  
4      male  35.0      0      0           373450   8.0500   NaN        S  
5      male   NaN      0      0           330877   8.4583   NaN        Q  
7      male   2.0      3      1           349909  21.0750   NaN        S  
12     male  20.0      0      0        A/5. 2151   8.0500   NaN        S  
..      ...   ...    ...    ...              ...      ...   ...      ...  
882  female  22.0      0      0             7552  10.5167   NaN        S  
884    male  25.0      0      0  SOTON/OQ 392076   7.0500   NaN        S  
885  female  39.0      0      5           382652  29.1250   NaN        Q  
888  female   NaN      1      2       W./C. 6607  23.4500   NaN        S  
890    male  32.0      0      0           370376   7.7500   NaN        Q  

[372 rows x 12 columns]

In [34]:
# Class 2 Survived --> 87
df[(df['Pclass'] == 2)  & (df['Survived']==1)]

PassengerId  Survived  Pclass  \
9             10         1       2   
15            16         1       2   
17            18         1       2   
21            22         1       2   
43            44         1       2   
..           ...       ...     ...   
831          832         1       2   
865          866         1       2   
866          867         1       2   
874          875         1       2   
880          881         1       2   

                                             Name     Sex    Age  SibSp  \
9             Nasser, Mrs. Nicholas (Adele Achem)  female  14.00      1   
15               Hewlett, Mrs. (Mary D Kingcome)   female  55.00      0   
17                   Williams, Mr. Charles Eugene    male    NaN      0   
21                          Beesley, Mr. Lawrence    male  34.00      0   
43       Laroche, Miss. Simonne Marie Anne Andree  female   3.00      1   
..                                            ...     ...    ...    ...   
831               Richards, Master. George Sibley    male   0.83      1   
865                      Bystrom, Mrs. (Karolina)  female  42.00      0   
866                  Duran y More, Miss. Asuncion  female  27.00      1   
874         Abelson, Mrs. Samuel (Hannah Wizosky)  female  28.00      1   
880  Shelley, Mrs. William (Imanita Parrish Hall)  female  25.00      0   

     Parch         Ticket     Fare Cabin Embarked  
9        0         237736  30.0708   NaN        C  
15       0         248706  16.0000   NaN        S  
17       0         244373  13.0000   NaN        S  
21       0         248698  13.0000   D56        S  
43       2  SC/Paris 2123  41.5792   NaN        C  
..     ...            ...      ...   ...      ...  
831      1          29106  18.7500   NaN        S  
865      0         236852  13.0000   NaN        S  
866      0  SC/PARIS 2149  13.8583   NaN        C  
874      0      P/PP 3381  24.0000   NaN        C  
880      1         230433  26.0000   NaN        S  

[87 rows x 12 columns]

In [35]:
#Class 2 died --> 97
df[(df['Pclass'] == 2)  & (df['Survived']==0)]

PassengerId  Survived  Pclass  \
20            21         0       2   
33            34         0       2   
41            42         0       2   
70            71         0       2   
72            73         0       2   
..           ...       ...     ...   
854          855         0       2   
861          862         0       2   
864          865         0       2   
883          884         0       2   
886          887         0       2   

                                                  Name     Sex   Age  SibSp  \
20                                Fynney, Mr. Joseph J    male  35.0      0   
33                               Wheadon, Mr. Edward H    male  66.0      0   
41   Turpin, Mrs. William John Robert (Dorothy Ann ...  female  27.0      1   
70                          Jenkin, Mr. Stephen Curnow    male  32.0      0   
72                                Hood, Mr. Ambrose Jr    male  21.0      0   
..                                                 ...     ...   ...    ...   
854      Carter, Mrs. Ernest Courtenay (Lilian Hughes)  female  44.0      1   
861                        Giles, Mr. Frederick Edward    male  21.0      1   
864                             Gill, Mr. John William    male  24.0      0   
883                      Banfield, Mr. Frederick James    male  28.0      0   
886                              Montvila, Rev. Juozas    male  27.0      0   

     Parch            Ticket  Fare Cabin Embarked  
20       0            239865  26.0   NaN        S  
33       0        C.A. 24579  10.5   NaN        S  
41       0             11668  21.0   NaN        S  
70       0        C.A. 33111  10.5   NaN        S  
72       0      S.O.C. 14879  73.5   NaN        S  
..     ...               ...   ...   ...      ...  
854      0            244252  26.0   NaN        S  
861      0             28134  11.5   NaN        S  
864      0            233866  13.0   NaN        S  
883      0  C.A./SOTON 34068  10.5   NaN        S  
886      0            211536  13.0   NaN        S  

[97 rows x 12 columns]

In [36]:
#class 1 --> Survived --> 136
df[(df['Pclass'] == 1)  & (df['Survived'] == 1)]

PassengerId  Survived  Pclass  \
1              2         1       1   
3              4         1       1   
11            12         1       1   
23            24         1       1   
31            32         1       1   
..           ...       ...     ...   
862          863         1       1   
871          872         1       1   
879          880         1       1   
887          888         1       1   
889          890         1       1   

                                                  Name     Sex   Age  SibSp  \
1    Cumings, Mrs. John Bradley (Florence Briggs Th...  female  38.0      1   
3         Futrelle, Mrs. Jacques Heath (Lily May Peel)  female  35.0      1   
11                            Bonnell, Miss. Elizabeth  female  58.0      0   
23                        Sloper, Mr. William Thompson    male  28.0      0   
31      Spencer, Mrs. William Augustus (Marie Eugenie)  female   NaN      1   
..                                                 ...     ...   ...    ...   
862  Swift, Mrs. Frederick Joel (Margaret Welles Ba...  female  48.0      0   
871   Beckwith, Mrs. Richard Leonard (Sallie Monypeny)  female  47.0      1   
879      Potter, Mrs. Thomas Jr (Lily Alexenia Wilson)  female  56.0      0   
887                       Graham, Miss. Margaret Edith  female  19.0      0   
889                              Behr, Mr. Karl Howell    male  26.0      0   

     Parch    Ticket      Fare Cabin Embarked  
1        0  PC 17599   71.2833   C85        C  
3        0    113803   53.1000  C123        S  
11       0    113783   26.5500  C103        S  
23       0    113788   35.5000    A6        S  
31       0  PC 17569  146.5208   B78        C  
..     ...       ...       ...   ...      ...  
862      0     17466   25.9292   D17        S  
871      1     11751   52.5542   D35        S  
879      1     11767   83.1583   C50        C  
887      0    112053   30.0000   B42        S  
889      0    111369   30.0000  C148        C  

[136 rows x 12 columns]

In [38]:
#class 1 --> Dead --> 80
df[(df['Pclass'] == 1)  & (df['Survived'] == 0)]

PassengerId  Survived  Pclass                                  Name  \
6              7         0       1               McCarthy, Mr. Timothy J   
27            28         0       1        Fortune, Mr. Charles Alexander   
30            31         0       1              Uruchurtu, Don. Manuel E   
34            35         0       1               Meyer, Mr. Edgar Joseph   
35            36         0       1        Holverson, Mr. Alexander Oskar   
..           ...       ...     ...                                   ...   
806          807         0       1                Andrews, Mr. Thomas Jr   
815          816         0       1                      Fry, Mr. Richard   
822          823         0       1       Reuchlin, Jonkheer. John George   
867          868         0       1  Roebling, Mr. Washington Augustus II   
872          873         0       1              Carlsson, Mr. Frans Olof   

      Sex   Age  SibSp  Parch    Ticket      Fare        Cabin Embarked  
6    male  54.0      0      0     17463   51.8625          E46        S  
27   male  19.0      3      2     19950  263.0000  C23 C25 C27        S  
30   male  40.0      0      0  PC 17601   27.7208          NaN        C  
34   male  28.0      1      0  PC 17604   82.1708          NaN        C  
35   male  42.0      1      0    113789   52.0000          NaN        S  
..    ...   ...    ...    ...       ...       ...          ...      ...  
806  male  39.0      0      0    112050    0.0000          A36        S  
815  male   NaN      0      0    112058    0.0000         B102        S  
822  male  38.0      0      0     19972    0.0000          NaN        S  
867  male  31.0      0      0  PC 17590   50.4958          A24        S  
872  male  33.0      0      0       695    5.0000  B51 B53 B55        S  

[80 rows x 12 columns]

# 3rd class Total 491
# 3rd class Survived --> 119
# 3rd class Dead --> 372
# ---------------------------
# Class 2 Survived --> 87
# Class 2 died --> 97
# ------------------------------
# class 1 --> Survived --> 136
# class 1 --> Dead --> 80

In [40]:
df[(df['Pclass'] == 1)  & (df['Survived'] == 0)].count()

PassengerId    80
Survived       80
Pclass         80
Name           80
Sex            80
Age            64
SibSp          80
Parch          80
Ticket         80
Fare           80
Cabin          59
Embarked       80
dtype: int64

In [41]:
df.head()

PassengerId  Survived  Pclass  \
0            1         0       3   
1            2         1       1   
2            3         1       3   
3            4         1       1   
4            5         0       3   

                                                Name     Sex   Age  SibSp  \
0                            Braund, Mr. Owen Harris    male  22.0      1   
1  Cumings, Mrs. John Bradley (Florence Briggs Th...  female  38.0      1   
2                             Heikkinen, Miss. Laina  female  26.0      0   
3       Futrelle, Mrs. Jacques Heath (Lily May Peel)  female  35.0      1   
4                           Allen, Mr. William Henry    male  35.0      0   

   Parch            Ticket     Fare Cabin Embarked  
0      0         A/5 21171   7.2500   NaN        S  
1      0          PC 17599  71.2833   C85        C  
2      0  STON/O2. 3101282   7.9250   NaN        S  
3      0            113803  53.1000  C123        S  
4      0            373450   8.0500   NaN        S

In [42]:
# Creating new clumns

In [46]:
df['my_Price'] = 100 + df['Fare']

In [47]:
df['my_Price']

0      107.2500
1      171.2833
2      107.9250
3      153.1000
4      108.0500
         ...   
886    113.0000
887    130.0000
888    123.4500
889    130.0000
890    107.7500
Name: my_Price, Length: 891, dtype: float64

In [48]:
df

PassengerId  Survived  Pclass  \
0              1         0       3   
1              2         1       1   
2              3         1       3   
3              4         1       1   
4              5         0       3   
..           ...       ...     ...   
886          887         0       2   
887          888         1       1   
888          889         0       3   
889          890         1       1   
890          891         0       3   

                                                  Name     Sex   Age  SibSp  \
0                              Braund, Mr. Owen Harris    male  22.0      1   
1    Cumings, Mrs. John Bradley (Florence Briggs Th...  female  38.0      1   
2                               Heikkinen, Miss. Laina  female  26.0      0   
3         Futrelle, Mrs. Jacques Heath (Lily May Peel)  female  35.0      1   
4                             Allen, Mr. William Henry    male  35.0      0   
..                                                 ...     ...   ...    ...   
886                              Montvila, Rev. Juozas    male  27.0      0   
887                       Graham, Miss. Margaret Edith  female  19.0      0   
888           Johnston, Miss. Catherine Helen "Carrie"  female   NaN      1   
889                              Behr, Mr. Karl Howell    male  26.0      0   
890                                Dooley, Mr. Patrick    male  32.0      0   

     Parch            Ticket     Fare Cabin Embarked  my_Price  
0        0         A/5 21171   7.2500   NaN        S  107.2500  
1        0          PC 17599  71.2833   C85        C  171.2833  
2        0  STON/O2. 3101282   7.9250   NaN        S  107.9250  
3        0            113803  53.1000  C123        S  153.1000  
4        0            373450   8.0500   NaN        S  108.0500  
..     ...               ...      ...   ...      ...       ...  
886      0            211536  13.0000   NaN        S  113.0000  
887      0            112053  30.0000   B42        S  130.0000  
888      2        W./C. 6607  23.4500   NaN        S  123.4500  
889      0            111369  30.0000  C148        C  130.0000  
890      0            370376   7.7500   NaN        Q  107.7500  

[891 rows x 13 columns]

In [49]:
df['Age_New'] = df['Pclass'] + df['Age']

In [50]:
df

PassengerId  Survived  Pclass  \
0              1         0       3   
1              2         1       1   
2              3         1       3   
3              4         1       1   
4              5         0       3   
..           ...       ...     ...   
886          887         0       2   
887          888         1       1   
888          889         0       3   
889          890         1       1   
890          891         0       3   

                                                  Name     Sex   Age  SibSp  \
0                              Braund, Mr. Owen Harris    male  22.0      1   
1    Cumings, Mrs. John Bradley (Florence Briggs Th...  female  38.0      1   
2                               Heikkinen, Miss. Laina  female  26.0      0   
3         Futrelle, Mrs. Jacques Heath (Lily May Peel)  female  35.0      1   
4                             Allen, Mr. William Henry    male  35.0      0   
..                                                 ...     ...   ...    ...   
886                              Montvila, Rev. Juozas    male  27.0      0   
887                       Graham, Miss. Margaret Edith  female  19.0      0   
888           Johnston, Miss. Catherine Helen "Carrie"  female   NaN      1   
889                              Behr, Mr. Karl Howell    male  26.0      0   
890                                Dooley, Mr. Patrick    male  32.0      0   

     Parch            Ticket     Fare Cabin Embarked  my_Price  Age_New  
0        0         A/5 21171   7.2500   NaN        S  107.2500     25.0  
1        0          PC 17599  71.2833   C85        C  171.2833     39.0  
2        0  STON/O2. 3101282   7.9250   NaN        S  107.9250     29.0  
3        0            113803  53.1000  C123        S  153.1000     36.0  
4        0            373450   8.0500   NaN        S  108.0500     38.0  
..     ...               ...      ...   ...      ...       ...      ...  
886      0            211536  13.0000   NaN        S  113.0000     29.0  
887      0            112053  30.0000   B42        S  130.0000     20.0  
888      2        W./C. 6607  23.4500   NaN        S  123.4500      NaN  
889      0            111369  30.0000  C148        C  130.0000     27.0  
890      0            370376   7.7500   NaN        Q  107.7500     35.0  

[891 rows x 14 columns]

In [51]:
df['Age_New'] = 100

In [52]:
df

PassengerId  Survived  Pclass  \
0              1         0       3   
1              2         1       1   
2              3         1       3   
3              4         1       1   
4              5         0       3   
..           ...       ...     ...   
886          887         0       2   
887          888         1       1   
888          889         0       3   
889          890         1       1   
890          891         0       3   

                                                  Name     Sex   Age  SibSp  \
0                              Braund, Mr. Owen Harris    male  22.0      1   
1    Cumings, Mrs. John Bradley (Florence Briggs Th...  female  38.0      1   
2                               Heikkinen, Miss. Laina  female  26.0      0   
3         Futrelle, Mrs. Jacques Heath (Lily May Peel)  female  35.0      1   
4                             Allen, Mr. William Henry    male  35.0      0   
..                                                 ...     ...   ...    ...   
886                              Montvila, Rev. Juozas    male  27.0      0   
887                       Graham, Miss. Margaret Edith  female  19.0      0   
888           Johnston, Miss. Catherine Helen "Carrie"  female   NaN      1   
889                              Behr, Mr. Karl Howell    male  26.0      0   
890                                Dooley, Mr. Patrick    male  32.0      0   

     Parch            Ticket     Fare Cabin Embarked  my_Price  Age_New  
0        0         A/5 21171   7.2500   NaN        S  107.2500      100  
1        0          PC 17599  71.2833   C85        C  171.2833      100  
2        0  STON/O2. 3101282   7.9250   NaN        S  107.9250      100  
3        0            113803  53.1000  C123        S  153.1000      100  
4        0            373450   8.0500   NaN        S  108.0500      100  
..     ...               ...      ...   ...      ...       ...      ...  
886      0            211536  13.0000   NaN        S  113.0000      100  
887      0            112053  30.0000   B42        S  130.0000      100  
888      2        W./C. 6607  23.4500   NaN        S  123.4500      100  
889      0            111369  30.0000  C148        C  130.0000      100  
890      0            370376   7.7500   NaN        Q  107.7500      100  

[891 rows x 14 columns]

In [54]:
df['Fare']

0       7.2500
1      71.2833
2       7.9250
3      53.1000
4       8.0500
        ...   
886    13.0000
887    30.0000
888    23.4500
889    30.0000
890     7.7500
Name: Fare, Length: 891, dtype: float64

In [55]:
# df['Fare'] = df['Age']

In [56]:
# df['New_Col'] # New_col Not present --> Creating columbn
# Write down New Column as Addition of Age + Pclass